recipes/tensorflow-lite/all/conandata.yml

```diff
@@ -2,11 +2,13 @@ sources:
   "2.6.0":
     url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.6.0.tar.gz"
     sha256: "41b32eeaddcbc02b0583660bcf508469550e4cd0f86b22d2abe72dfebeacde0f"
+  "2.6.2":
+    url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.6.2.tar.gz"
+    sha256: "e68c1d346fc3d529653530ca346b2c62f5b31bd4fcca7ffc9c65bb39ab2f6ed3"
+  "2.9.1":
+    url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.9.1.tar.gz"
+    sha256: "6eaf86ead73e23988fe192da1db68f4d3828bcdd0f3a9dc195935e339c95dbdc"
 patches:
-  "2.6.0":
-    - patch_file: "patches/local_find_packages.patch"
-      base_path: "src"
-    - patch_file: "patches/link_targets.patch"
-      base_path: "src"
-    - patch_file: "patches/non_x86_support.patch"
+  "2.9.1":
+    - patch_file: "patches/msvc_fixes.patch"
       base_path: "src"
```

Comment:
This looks out of place, https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#subfolder-properties 

***

UPDATE: dont change this it will go away with v2 migration